sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -89,6 +89,22 @@ def analyze_custom_documents(custom_model_id):
                     print(f"...{region.page_number}")
             for cell in table.cells:
                 print(f"...Cell[{cell.row_index}][{cell.column_index}] has text '{cell.content}'")
+
+    # Extract table cell values from "documents" field in result and print the name/value information
+    if result.documents:
+        for idx, doc in enumerate(result.documents):
+            print(f"\n{doc.fields.__len__()} fields found on Document {idx}" )
+            for fieldName,fieldValue in doc.fields.items():
+                # "TableLayout" is the table field name which you labeled. Table cell information store as array in documnet field.
+                if fieldName=="TableLayout" and fieldValue.type=="array" and fieldValue.value_array:
+                    print(f"------Start Extracting Table Cell Values------")
+                    for rowIdx, obj in enumerate(fieldValue.value_array):
+                        valObj= obj["valueObject"]
+                        if valObj:
+                            for key in valObj.keys():
+                                print(f"...RowIndex:{rowIdx}, ColName: {key}, CellValue: {getattr(valObj[key], "content")}")
```

Comment:
can't we use `getattr(valObj[key], "content")` => `valObj[key].content`

Reply:
optimized